conn.go

```diff
@@ -160,6 +166,18 @@ func (c *Conn) TLSConnectionState() (state tls.ConnectionState, ok bool) {
 	return tc.ConnectionState(), true
 }
 
+func (c *Conn) Info() (info ConnInfo) {
```

Comment:
Naked returns should be avoided if possible, when they don't convey a value for documentation: https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
Fixed.